releasenotes/notes/metrics-api-5fb1d572d7f7137e.yaml

```diff
@@ -0,0 +1,5 @@
+---
+other:
+  - |
+    Remove references to runtime metrics from the Tracer object and into its own
+    API, similarly to what is done for the profiling service.
```

Comment:
I think we should have an `upgrade` entry here with instruction on how to enable/configure runtime metrics for those who were using `tracer.configure(collect_metrics=True)`